app/profile/components/SaveCustomToken.js

```diff
@@ -0,0 +1,55 @@
+// @flow strict
+
+import * as React from 'react';
+import {
+  Text,
+  TextInput,
+  Button,
+  StyleSheet,
+  Color,
+} from '@kiwicom/mobile-shared';
+import { Translation } from '@kiwicom/mobile-localization';
+
+type Props = {|
+  +onPress: (token: string) => void,
+|};
+
+type State = {|
+  token: string,
+|};
+
+export default class SaveCustomToken extends React.Component<Props, State> {
+  state = {
+    token: '',
+  };
+
+  onChangeText = (token: string) => {
+    this.setState({ token });
+  };
+
+  onPress = () => {
+    this.props.onPress(this.state.token);
+  };
+
+  render = () => {
+    return (
+      <React.Fragment>
+        <TextInput
+          placeholder={<Translation passThrough="Custom token" />}
```

Comment:
So since it's only for testing purposes, we don't need any translations as it won't stay in the app forever, right?

Reply:
Exactly